react/features/large-video/components/LargeVideo.web.js

```diff
@@ -70,6 +72,9 @@ export default class LargeVideo extends Component<*> {
                             muted = { true } />
                     </div>
                 </div>
+                { APP.store.getState()['features/base/config']
```

Comment:
Don't use the global store when the configuration can be obtained through redux. If config is going to be used, config can be obtained through redux without a global, be it by connecting LargeVideo or passing it in through Conference.web. The global store is used for those places that don't have ready access to the store and are in the never-ending purgatory of waiting to get converted to react/redux.

Reply:
I will move the `disableTranscriptionSubtitles` toggle to interfaceConfig. I guess that cannot be accessed via redux, right ?